packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -498,15 +497,20 @@ mochaSuiteFn('opentelemetry/instrumentations', function () {
         ));
   });
 
-  // Skipping the tests on CI for now, because there is a existing timeout issue while connectiong the azure sql,
-  // see github issue https://github.com/tediousjs/tedious/issues/1277
-  const runTedious = isCI() ? describe.skip : describe;
+  const runTedious = describe;
   runTedious('tedious', function () {
```

Comment:
```suggestion
  describe('tedious', function () {
```

Suggestion: There is no need to save mocha's `describe` into a new variable called `runTedious` anymore.

Reply:
Sure. Will update